presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -29,6 +29,7 @@
     private Optional<String> glueRegion = Optional.empty();
     private Optional<String> glueEndpointUrl = Optional.empty();
     private boolean pinGlueClientToCurrentRegion;
+    private int maxGlueErrorRetries = 10;
```

Comment:
is `10` default glue value?

Reply:
Default is no retries.  I used 10 because that's the same default Presto sets for `hive.s3.max-error-retries` (which sets the same property on AWS's `ClientConfiguration`)